### DISCUSSION TITLE
Which Comes first?

### DISCUSSION THREAD

**Discussion by thedinz** [Taxel/PlexTraktSync#1150] [Q&A]
Does PTS check plex or trakt first?

It seems like the latter, and if so, can that be changed? It seems like once i mark something watched in plex, it syncs with trakt and if i then later one mark it as un-watched in plex next time PTS runs it flips it back to watched.

**Comment by simonc56** [comment]
#1080

**Comment by glensc** [comment]
stupid title. i guess answer is egg. or is it chicken?
